src/scripting/lua_map.cc

```diff
@@ -5443,6 +5444,27 @@ int LuaBuilding::dismantle(lua_State* L) {
 	return 0;
 }
 
+/* RST
+   .. method:: enhance([keep_wares = false])
+
+      versionadded:: 1.1
```

Comment:
Almost. RST markup takes some getting used to…
```suggestion
      .. versionadded:: 1.1
```